src/gui/gtk.c

```diff
@@ -2667,6 +2667,32 @@ void dt_gui_load_theme(const char *theme)
     g_clear_error(&error);
   }
 
+  if(dt_conf_get_bool("themes/usertweaks"))
+  {
+    g_snprintf(usercsspath, sizeof(usercsspath), "%s/usertweaks.css", configdir);
```

Comment:
I suppose that the file isn't in the `theme` subdir to avoid mixing with "real" themes. Right ?
So why no using "just" `user.css` ? it may be simpler to remember and less prone to typing errors...

Reply:
Can do. Naming things is always where I run out of ideas.

And yes I've kept it out of themes on purpose because it needs a theme to be loaded as well.

Reply:
Also the description/tooltip in the preferences might need some editing. Again, I wasn't entirely sure how much to put.